app/models/location.rb

```diff
@@ -1,7 +1,17 @@
 class Location < ActiveRecord::Base
+  SUPPORTED_ZIPCODES = %w[80221].freeze
+
   belongs_to :user, touch: true
 
   validates :address, presence: true
   validates :user, presence: true
   validates :zipcode, presence: true
+
+  def supported?
+    SUPPORTED_ZIPCODES.include?(zipcode)
+  end
+
+  def zipcode=(zipcode)
+    super(zipcode.to_s.strip)
```

Comment:
I'd prefer to keep it in the model for the sake of ensuring correctness regardless of the data's origins.

@joshuaclayton @kenyonj are either of you strongly opposed to that?